app/styles/_list-pf.less

```diff
@@ -28,7 +28,6 @@
   .dropdown-kebab-pf {
     .dropdown-toggle{
       font-size: 18px;
-      margin-right: -5px;
```

Comment:
This change results in the dropdown caret and the kebab being misaligned.

![screen shot 2018-01-11 at 11 15 45 am](https://user-images.githubusercontent.com/895728/34834769-da514926-f6c0-11e7-8253-dc48521db625.PNG)

Reply:
Corrected alignment
<img width="195" alt="screen shot 2018-01-11 at 3 30 51 pm" src="https://user-images.githubusercontent.com/1874151/34850181-d2f3253c-f6f2-11e7-97f9-b5af9b4ad182.png">